tests/integration/src/test/java/org/apache/pulsar/tests/integration/functions/java/PulsarFunctionsJavaTest.java

```diff
@@ -166,7 +166,7 @@ public void testSlidingCountWindowTest() throws Exception {
     @Test(groups = {"java_function", "function"})
     public void testMergeFunctionTest() throws Exception {
 	    testMergeFunction();
-   }
+    }
```

Comment:
nit: remove space

Reply:
This is to align with the rest of the function on 4 spaces.
But this whole class is misaligned...

Reply:
I realigned completely this block but I guess the full class should be realigned.
Maybe better to do it in another PR ?

Reply:
yes, this PR should focus on the feature and not on code clean up